modules/base/admin/module.py

```diff
@@ -204,8 +204,21 @@ async def repository_install(self, ctx, url: str, branch: Optional[str] = None):
     @commands.max_concurrency(1, per=commands.BucketType.default, wait=False)
     @commands.check(check.acl)
     @repository.command(name="update", aliases=["fetch", "pull"])
-    async def repository_update(self, ctx, name: str):
-        """Update module repository."""
+    async def repository_update(self, ctx, name: str, option: str = None):
+        """Update module repository.
+
+        Args:
+            name: Repository name
+            option: Optional update type (FORCE = force pull, RESET = hard reset)
+        """
+        if option:
+            option = option.upper()
+            if option not in ["RESET", "FORCE"]:
```

Comment:
Why are they uppercase?

Reply:
I liked this more, but they can be OC lowercase, just has to switch it here and replace `.upper()` with `.lower()`. Does it matter?

Reply:
It's a matter of consistency. All other modifiers we use are lowercase.

Reply:
Ok, fixed.